docs/cli-usage.md

```diff
@@ -33,7 +33,9 @@ Flags:
 
  - ```--span-size``` : Span size that soci index uses to segment layer data. Default is 4MiB 
  - ```--min-layer-size``` : Minimum layer size to build zTOC for. Smaller layers won't have zTOC and not lazy pulled. Default is 10MiB
- - ```--disable-xattrs``` : When true, adds DisableXAttrs annotation to SOCI index. This annotation often helps performance at pull time. Default is true.
+ - ```--optimizations``` : Enable experimental features by name. Usage is `--optimizations opt_name`.
```

Comment:
These are not part of the initial PR, but still coming up in the revert. intentional?

Reply:
The original PR never added documentation, so I just added it here.